_episodes/02-day1.md

```diff
@@ -19,7 +19,72 @@ keypoints:
+
+Raster data is any pixelated (or gridded) data where each pixel is associated with a specific geographic location. The value of a pixel can be continuous (e.g. elevation) or categorical (e.g. land use). This data structure is very common: it's how we represent any digital image. A geospatial raster is only different from a digital photo in that it is accompanied by spatial information that connects the data to a particular location. This includes the raster's extent and cell size, the number of rows and columns, and its coordinate reference system (CRS). 
+
+Some examples of continuous rasters include:
+
+1. Precipitation maps
+2. Maps of tree height derived from LiDAR data 
+3. Elevation values for a region
+
+Some rasters contain categorical data where each pixel represents a discrete class such as landcover type (e.g., "forest" or "grassland") rather than a continuous value such as elevation or temperature. Some examples of classified maps include:
+
+1. Landcover/land use maps
+2. Tree height maps classified as short, medium, and tall trees
+3. Elevation maps classified as low, medium, and high elevation
+
+> ## Advantages and Disadvantages
+> 
+> With your neighbor, brainstorm potential advantages and disadvantages of storing data in raster format. 
+> 
+> > ## Solution
+> > 
+> > Raster data has some important advantages:
+> > 
+> > * representation of continuous surfaces
+> > * potentially very high levels of detail
+> > * data is 'unweighted' across its extent - the geometry doesn't implicitly highlight features
+> > * cell-by-cell calculations can be very fast and efficient
+> > 
+> > The downsides of raster data are:
+> > 
+> > * file sizes become very large as the cell size gets smaller (more pixels)
+> > * currently popular formats don't embed metadata well
+> > * can be difficult to represent complex information
+> {: .solution}
+{: .challenge}
+
+## Important Attributes of Raster Data
+
+### Extent
+
+The spatial extent is the geographic area that the raster data covers. The spatial extent of an R spatial object represents the geographic edge or location that is the furthest north, south, east, and west. In other words, extent represents the overall geographic coverage of the spatial object. 
+
+### Resolution
+
+A resolution of a raster represents the area on the ground that each pixel of the raster covers.
+
+## Raster Data Format for this Capstone
+
+Raster data can come in many different formats. For this capstone, we've offered data in the GeoTIFF format, which has the extension `.tif`. A `.tif` file stores metadata or attributes about the file as embedded `.tif tags`. For instance, your camera might store a tag that describes the make and model of the camera or the date that the photo was taken when it saves a `.tif`. A GeoTIFF is a standard `.tif` image format with additional spatial (georeferencing) information embedded in the file as tags. These tags should include the following raster metadata:
```

Comment:
It may be worth mentioning that GeoTIFF and a regular, non-georeferenced TIFF share the same file extension